test/baseline.spec-cpp.sh

```diff
@@ -0,0 +1,20 @@
+#! /bin/bash
+
+
+#TODO(Jesse): Maybe make another results file for the failures?
+
+output_filename=./baseline.spec-cpp.results
+
+[ -f $output_filname ] && rm $output_filename
```

Comment:
You should add our header to this script, also called "unofficial bash strict mode":

```
set -o errexit
set -o pipefail
set -o nounset
```

Because there is a typo in the "filname" variable and I think that's why the file has twice the number of reuslts I'd expect!

(Oil has this on by default, that was the subject of this blog post: https://www.oilshell.org/blog/2022/05/release-0.10.0.html)

Reply:
Fixed